.github/workflows/e2e.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: '*'
```

Comment:
was this just to ensure it was triggered and should be brought back? I'm bad at gh actions so idk what the implications of this are

Reply:
`branches` was used as a filter. Not sure why "*" wasn't enough, but by removing it we're saying "always run e2e on PRs"

Reply:
so this line should stay removed